File: Samples/Objective-C/nhubsample-refresh/nhubsample-refresh/NotificationDetailsViewController.m
//----------------------------------------------------------------
//  Copyright (c) Microsoft Corporation. All rights reserved.
//----------------------------------------------------------------

#import "NotificationDetailsViewController.h"

@interface NotificationDetailsViewController ()

@end

@implementation NotificationDetailsViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    
    self.summaryLabel.text = self.notification.title;
    self.detailsLabel.text = self.notification.body;
    
    [self.detailsLabel sizeToFit];
}

@end

File: Samples/Swift/nhubtestapp/nhubtestapp/NotificationDetailsViewController.swift
//
//  NotificationDetailsViewController.swift
//  nhubtestapp
//
//  Created by Artem Egorov on 4/23/20.
//  Copyright © 2020 Microsoft. All rights reserved.
//

import UIKit

class NotificationDetailsViewController: UIViewController {
    @IBOutlet weak var summaryLabel: UILabel!
    @IBOutlet weak var detailsLabel: UILabel!
    
    var notification = ""
    
    override func viewDidLoad() {
        super.viewDidLoad()
        summaryLabel.text = notification
        detailsLabel.text = "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
        detailsLabel.sizeToFit()
        // Do any additional setup after loading the view.
    }
    

    /*
    // MARK: - Navigation

    // In a storyboard-based application, you will often want to do a little preparation before navigation
    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
        // Get the new view controller using segue.destination.
        // Pass the selected object to the new view controller.
    }
    */

}
